README.md

```diff
@@ -19,6 +19,9 @@ Build:
 - Git
 - JDK11+
 - Gradle
+- Docker
```

Comment:
Docker is no longer used for the image builds actually, I just recently changed it to use podman.

Reply:
Ah shoot; I did a git pull to check for changes but forgot the default was my fork, not upstream...

My bad.

Reply:
I've force-pushed an updated commit that specifies Podman instead.